### DISCUSSION TITLE
coredns throwing readiness error 503

### DISCUSSION THREAD

**Discussion by ritik872000** [aws/amazon-vpc-cni-k8s#2997] [Q&A]
So, I have enabled custom networking with secondary CIDR but my coredns pods are not coming up even with snat parameter tru I have been scratching  my head for a week now to resolve this issue.

**Comment by hkhelif** [comment]
@ritik872000 can you provide the output of 

- `kubectl describe pod -n kube-system coredns-PODNAMEHERE`
- Enable coredns logs as [mentioned here](https://repost.aws/knowledge-center/eks-dns-failure) then run `kubectl logs -n kube-system coredns-PODNAMEHERE`

**Comment by ritik872000** [comment]
<img width="739" alt="image" src="https://github.com/user-attachments/assets/fce9290b-5011-421a-b97e-608be8232db6">
